setup.py

```diff
@@ -7,7 +7,7 @@
 """
 import pathlib
 
-from setuptools import setup
+from setuptools import setup, find_packages
```

Comment:
Small change, but can you swap the order of these two imports to keep them alphabetical?